docs/pages/jpsreport/jpsreport_fd.md

```diff
@@ -0,0 +1,13 @@
+---
+title:  Fundamental diagram 
+keywords: analysis
+tags: [jpsreport, jpsreport_tutorial]
+sidebar: jupedsim_sidebar
+folder: jpsreport
+summary: This example shows how to calculated the fundamental diagram using jpsreport
+permalink: jpsreport_fd.html
```

Comment:
I would remove this page for this PR, see issue #629

Reply:
Is it enough to remove it from the sidebar, so that it becomes "invisible", and mark it as "unpublished"?
Or do you mean delete it for good?